manifests/init.pp

```diff
@@ -1,27 +1,27 @@
-# Class: bareos
-#
-# Authors
-# -------
-#
-# Richard Hillmann <richie@project0.de>
+# This class manages the bareos repository
+# Parameters should be configured in the upper class `::bareos`.
```

Comment:
were those comments meant to be in the init.pp?

Reply:
c&p fail. I restored the old header.